plugin/phpactor.vim

```diff
@@ -0,0 +1,4 @@
+augroup phpactor
+  autocmd!
+  autocmd FileType php call phpactor#project#bufferMatcher#assignFileToProject(expand('<afile>'))
```

Comment:
Why not put this in a filetype plugin file instead of using an auto command ?

Reply:
`ftplugin/php.vim` is prevented from be sourced more than once. 

Maybe we can put it in another ftplugin file. 

Thanks to your above suggestion I discovered that `ftplugin/php/commands.vim` has no such prevention (commands are redefined each time).

Reply:
> ftplugin/php.vim is prevented from be sourced more than once.

Why do we care about it being resourced? btw I use https://github.com/tpope/vim-scriptease to  resource plugins in development `:Runtime!`

Reply:
IMHO, part of mentioned above command definitions (as `PhpactorClassNew`, with given project root, or `PhpactorExtensionInstall`) would be moved from ftplugin to plugin.

Reply:
> ftplugin/php.vim is prevented from be sourced more than once.

I don't get that, it will be executed for each time a buffer filetype is defined as `php`.

> Thanks to your above suggestion

I think you refer to the `buffer` part for commands.
We should try to only impact one buffer at a time since we only care for php files (well behat too but that's not the point).

Reply:
https://github.com/phpactor/phpactor/blob/faecc433e98060dba1641fefa712eb3f8ceb43ef/ftplugin/php.vim#L1-L3

That's why it can be sourced only once.
I made a PR to clean the filetype plugin a bit:
https://github.com/phpactor/phpactor/pull/964